### DISCUSSION TITLE
can't use shift+insert to paste text in deepin-terminal

### DISCUSSION THREAD

**Discussion by chenzhiwei** [linuxdeepin/developer-center#3020] [Features Request & Ideas | 特性请求 & 头脑风暴]
can't use shift+insert to paste text in deepin-terminal

**Comment by ch593030323** [comment]
Thank u for your suggestion. We will consider this function reasonably.

**Comment by chenzhiwei** [comment]
@ch593030323 this is not a suggestion, I can't set `shift+insert` as paste shortcut in deepin-terminal settings.

**Comment by ch593030323** [comment]
Before i saw ur the PR in "https://github.com/linuxdeepin/deepin-terminal/pull/48".
Others terminals are supported the function.
i will submit the issue later, please wait for the reply with me.

**Comment by chenzhiwei** [comment]
Not only terminals, but also the GUI applications, such as Chrome, Firefox.

You can try to copy some text and use `shift+insert` to paste in Chrome or Firefox.

**Comment by meiyixiang** [comment]
Thank you for your feedback. I have recorded this problem and fed back to product colleagues

**Comment by meiyixiang** [comment]
Hello, what version of the community are you using now? Our latest version of the terminal is to support this function you said please upgrade the system

**Comment by chenzhiwei** [comment]
please close this discussion.

**Reply by BLumia** [reply]
Discussion cannot (and doesn't need to) be closed, so I'll mark this as resolved then :)

**Comment by meiyixiang** [comment]
This suggestion is accepted, and optimization will be scheduled later
